codec-http2/src/test/java/io/netty/handler/codec/http2/InboundHttp2ToHttpAdapterTest.java

```diff
@@ -265,7 +268,7 @@ public void clientRequestOneDataFrame() throws Exception {
                 @Override
                 public void run() {
                     frameWriter.writeHeaders(ctxClient(), 3, http2Headers, 0, false, newPromiseClient());
-                    frameWriter.writeData(ctxClient(), 3, content.retain(), 0, true, newPromiseClient());
+                    frameWriter.writeData(ctxClient(), 3, content.duplicate().retain(), 0, true, newPromiseClient());
```

Comment:
Otherwise equality comparisons fail.  With the new `Local*Channel` the ByteBuf indexes are updated before the mockito matchers are applied and our output is not captured because we have a eq(..) comparison for the bytebuf argument on the matchers.